openedx/core/lib/api/view_utils.py

```diff
@@ -92,7 +92,10 @@ def _decorator(func_or_class):
         Requires either OAuth2 or Session-based authentication.
         If is_user is True, also requires username in URL matches the request user.
         """
+        from edx_rest_framework_extensions.authentication import JwtAuthentication
```

Comment:
Numerous apps load the user model on startup. There is something wrong here in this file/project.